largest_palindrome_product.js

```diff
@@ -5,18 +5,42 @@
  *  @return {Object} an object containing the two factors used to produce
  *                   the palindromeNumber and the palindromeNumber itself.
  */
-module.exports = function(digits){
+module.exports = function(digits) {
   var factor_0 = 0;
   var factor_1 = 0;
   var palindromeNumber = 0;
 
   // do your work here
 
+  function checkPalindrome(num) {
+    let numToString = num.toString();
+    const numOfDigits = numToString.length;
+    for (let i = 0; i < Math.floor(numOfDigits / 2); i++) {
+      if (
+        numToString.substring(i, i + 1) !==
+        numToString.substring(numOfDigits - i - 1, numOfDigits - i)
+      ) {
+        return false;
+      }
+    }
+    return true;
+  }
 
+  for (let i = 10 ** (digits - 1); i < 10 ** digits; i++) {
+    for (let j = 10 ** (digits - 1); j < 10 ** digits; j++) {
```

Comment:
Approving this brute force way of checking for the largest palindromic product but can you think of a way to optimize the inner for loop?